flarefly/fitter.py

```diff
@@ -961,7 +961,7 @@ def get_significance(self, idx=0, nsigma=3):
             (signal[1]**2 + bkg[1]**2) / (4. * sig_plus_bkg**2) + (
                 bkg[0]/sig_plus_bkg) * signal[1]**2 / signal[0]**2)
 
-        return significance, significance_err
+        return float(significance), float(significance_err)
```

Comment:
There should not be need since both signal and background returned by `get_signal` and `get_background` now are floats
```suggestion
        return significance, significance_err
```